HW_lesson6/classes.py

```diff
@@ -0,0 +1,26 @@
+class Circle:
+
+    X = 10
```

Comment:
move this variables to the __init__ and make them as an instance variables and as input parameters to the init method